lib/add_to_calendar.rb

```diff
@@ -27,6 +27,62 @@ def initialize(start_datetime:, end_datetime: nil, title:, timezone:, location:
   
       validate_attributes
     end
+
+    def hey_url
+      calendar_url = "https://app.hey.com/calendar/ical_events/new?ical_source=BEGIN%3AVCALENDAR%0ABEGIN%3AVEVENT"
+      
```

Comment:
@jorgemanrubia  Ty! moved too fast, updated the initial calendar URL moving out the prodid.